extensions/wikia/WikiFactory/tests/WikiFactoryTest.php

```diff
@@ -223,6 +223,23 @@ public function testPrepareUrlToParseDataProvider() {
 		];
 	}
 
+	public function testGetHostById() {
+		$this->mockStaticMethod( 'WikiFactory', 'getVarValueByName', 1 );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiFactory" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)